its/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "sonarlint-its",
+  "displayName": "sonarlint-its",
+  "description": "SonarLint ITs",
+  "version": "0.0.1",
+  "engines": {
+    "vscode": "^1.41.0"
```

Comment:
Should this be `1.36.0` for consistency with the extension under test?

Reply:
Yep, but this is surprising the IT axis 1.36.0 was passing...